relays/bin-substrate/src/cli/register_parachain.rs

```diff
@@ -83,6 +83,11 @@ macro_rules! select_bridge {
 
 				use bp_rialto::{PARAS_PALLET_NAME, PARAS_REGISTRAR_PALLET_NAME};
 
+				const RELAY_CHAIN_SPEC_VERSION: u32 = rialto_runtime::VERSION.spec_version;
+				const RELAY_CHAIN_TRANSACTION_VERSION: u32 = rialto_runtime::VERSION.spec_version;
+				const PARA_CHAIN_SPEC_VERSION: u32 = rialto_runtime::VERSION.spec_version;
+				const PARA_CHAIN_TRANSACTION_VERSION: u32 = rialto_runtime::VERSION.spec_version;
```

Comment:
I'm not sure it is correct.

Reply:
Could you, please, use `rialto_parachain_runtime::VERSION.spec_version` here?

Reply:
It fixed here https://github.com/paritytech/parity-bridges-common/pull/1248/commits/91c9d51a3ed08c448f31fdad3f4ff668120326b6 please check it